hyp3lib/rtc2color.py

```diff
@@ -1,15 +1,40 @@
-"""Converts a dual-pol RTC to a color GeoTIFF"""
+"""RGB decomposition of a dual-pol RTC
 
-from __future__ import print_function, absolute_import, division, unicode_literals
+The RGB decomposition enhances RTC dual-pol data for visual interpretation. It
+decomposes the co-pol and cross-pol signal into these color channels:
+    red: simple bounce (polarized) with some volume scattering
+    green: volume (depolarized) scattering
+    blue: simple bounce with very low volume scattering
+
+In the case where the volume to simple scattering ratio is larger than expected
+for typical vegetation, such as in glaciated areas or some forest types, a teal
+color (green + blue) is used.
+"""
 
 import argparse
 import os
+from pathlib import Path
+from typing import Union
+
 import numpy as np
 from osgeo import gdal, osr
 
 
-def rtc2color(fullpolFile, crosspolFile, threshold, geotiff, cleanup=False,
-  teal=False, amp=False, real=False):
+def rtc2color(copol_tif: Union[str, Path], crosspol_tif: Union[str, Path], threshold: float, out_tif: Union[str, Path],
+              cleanup=False, teal=False, amp=False, real=False):
+  """RGB decomposition of a dual-pol RTC
+
+  Args:
+      copol_tif: Path to the co-pol GeoTIF
+      crosspol_tif: Path to the cross-pol GeoTIF
+      threshold: Decomposition threshold in db
+      out_tif: Path to the output GeoTIF
+      cleanup: Cleanup bad data using a -24 db threshold for valid pixels
+      teal: Combine green and blue channels because the volume to simple scattering ratio is high
+      amp: input TIFs are in amplitude and not power
+      real: Output floating point values instead of RGB scaled (0--255) ints
+  """
+  # FIXME: Can we just determine if we should use teal?
```

Comment:
Documentation says we use teal 
> in the case where the volume to simple scattering ratio is larger than expected for typical vegetation, such as in glaciated areas or some forest types
However, we are always using `teal=True`: https://github.com/asfadmin/cloud-proj/blob/c5dda50e5defafa473df1f5cbe89d856f8daa239/processing/proc/proc_rgb.py#L145

Could we just determine in the ratio is too high (what does that mean?) and decide on teal in this fn?

Reply:
Importantly, when we call this from RTC GAMMA, RTC SNAP, or Geocode Only, `teal=False` but is always `teal=True` in the RGB Decomp. process.

Reply:
Also associated with `teal`, `teal` and `real` are *way too similar*.

Reply:
Consensus is `teal` option should generally be benign as the changes it introduces in most scenes are inconsequential and inst. knowledge of this option departed when creator left ASF. I'm going to leave this alone for now and open an issue for v2